.github/workflows/ci.yml

```diff
@@ -186,3 +186,33 @@ jobs:
         run: ./mvnw -B clean install -Dmaven.javadoc.skip=true -Drat.skip=true -Dcheckstyle.skip=true -DskipITs -DskipTests -Dspotless.apply.skip=true
       - name: Run Example generator
         run: echo y | ./mvnw -B test -f examples/shardingsphere-example-generator/pom.xml -Pexample-generator
+
+  test-coverage-report:
+    runs-on: ubuntu-latest
+    name: shardingsphere test-coverage report
+    steps:
+      - uses: actions/checkout@v2
+      - name: Maven resolve ranges
+        run: ./mvnw versions:resolve-ranges -ntp -Dincludes='org.springframework:*,org.springframework.boot:*'
+      - name: Cache Maven Repos
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+#      - name: Shutdown Ubuntu MySQL(SUDO)
+#        run: sudo service mysql stop
+#      - name: Set up MySQL
+#        uses: mirromutth/mysql-action@v1.1
+#        with:
+#          mysql root password: "root"
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: 8
+      - name: Test with Maven
+        run: ./mvnw -B --no-transfer-progress clean install cobertura:cobertura -Dmaven.javadoc.skip=true -Ddatabases=MySQL,PostgreSQL
```

Comment:
`-Ddatabases=MySQL,PostgreSQL` is for IT only, could we consider about remove the arguments?